.eslintignore

```diff
@@ -39,6 +39,33 @@ packages/lit-element/lit.min.*
 packages/lit-element/platform-support.*
 packages/lit-element/private-ssr-support.*
 
+packages/lit-element-starter-ts/node_modules/
+packages/lit-element-starter-ts/lib/
+packages/lit-element-starter-ts/test/
+
+# top level source
+packages/lit-element-starter-ts/**/my-element.js
+packages/lit-element-starter-ts/**/my-element.js.map
+packages/lit-element-starter-ts/**/my-element.d.ts
+packages/lit-element-starter-ts/**/my-element.d.ts.map
+
+packages/lit-element-starter-ts/node_modules/*
```

Comment:
Is this needed since your'e ignoring the whole folder?

Reply:
This is a generated file (by `npm run ignore-sync`); since `packages/lit-element-starter-ts/.eslintignore` has a (package) top-level `node_modules/*` ignore, that gets turned into this at the monorepo root. It's fine I think.